tests/e2e/rwx_topology_utils.go

```diff
@@ -942,3 +933,204 @@ func fetchListofHostsInCluster(ctx context.Context, clusterName string) []string
 	}
 	return hostList
 }
+
+/*
+writeNewDataAndUpdateVsphereConfSecret uitl edit the vsphere conf and returns the updated
+vsphere config sceret
+*/
+func writeNewDataAndUpdateVsphereConfSecretForFileVolumes(client clientset.Interface, ctx context.Context,
```

Comment:
rename it

Reply:
done, new name - updateVsphereConfSecretForFileVolumes